File: OOP/Lab_5_C++/Ball.cpp
//
// Created by Den on 12.11.2016.
//

#include "Ball.h"
Ball::Ball() {
    r=0;
}
Ball::Ball(double radius) {
    r=radius;
}
double Ball::area() {
    return r*r*4*3.14;
}
double Ball::volume() {
    return 4.0/3.0*r*r*r*3.14;
}

File: OOP/Lab_5_Java/src/abstr/Ball.java
package abstr;

/**
 * Created by Den on 05.12.2016.
 */
public class Ball extends Body{
    public double r;
    Ball() {
        r=0;
    }
    Ball(double radius) {
        r=radius;
    }
    public double area() {
        return r*r*4*3.14;
    }
    public double volume() {
        return 4.0/3.0*r*r*r*3.14;
    }
}
